web/client/components/map/openlayers/swipe/SliderSwipeSupport.jsx

```diff
@@ -22,20 +25,41 @@ const VSlider = ({ type, map, widthRef }) => {
     };
 
     return (
-        <Draggable axis="x" bounds="parent" onDrag={(e, ui) => onDragVerticalHandler(e, ui)}>
+        <Draggable
+            axis="x"
+            bounds="parent"
+            onStart={() => setShowArrows(false)}
+            onDrag={(e, ui) => onDragVerticalHandler(e, ui)}
+            onStop={() => setShowArrows(true)}>
             <div className="mapstore-swipe-slider" style={{
                 height: "100%",
                 top: '0px',
                 left: `${map.getProperties().size[0] / 2}px`,
-                width: "12px",
+                width: "8px",
                 cursor: "col-resize"
-            }}></div>
+            }}>
+                {showArrows && (
+                    <div className="ms-vertical-swipe-slider-arrows" style={{
+                        // subtract half of the arrows height i.e 52px so that top stops at center
+                        top: `${(map.getProperties().size[1] / 2) - 26}px`
+                    }}>
+                        <Button className="square-button-md">
+                            <Glyphicon glyph="chevron-left" />
+                        </Button>
+                        <Button className="square-button-md">
+                            <Glyphicon glyph="chevron-right" />
+                        </Button>
```

Comment:
You shouldn't use buttons, just normal div (the also have some visual feedback for click, but they are not click-able)
I see they are draggable, so maybe the cursor should be the same of the drag handle (n-resize, w-resize or what it is) if possible, but this is a plus
![Peek 07-10-2020 12-44](https://user-images.githubusercontent.com/1279510/95321206-01517200-089b-11eb-8ef0-2c462df4a07a.gif)
.